stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -372,6 +376,31 @@ private void echo(String msg) {
         log.info(msg);
     }
 
+    private void copyUploadDirectoryToAgents() throws IOException {
+        log.info("WORKING DIRECTORY:" + WORKING_DIRECTORY);
+        if (!UPLOAD_DIRECTORY.exists()) {
+            log.debug("Skipping upload, since no upload file in working directory");
+            return;
+        }
+        log.info("UPLOAD DIRECTORY:" + UPLOAD_HOME);
+        List<File> files = Utils.getFilesFromClassPath(UPLOAD_HOME);
+        for (String ip : agentsClient.getPublicAddresses()){
+            for (File file : files){
+                String syncCommand = format("rsync -avv -e \"ssh %s\" %s %s@%s:hazelcast-stabilizer-%s/workers/%s/",
```

Comment:
Add an info statement either before or after with:

 log.info(format("Starting upload '+%s+' to agent %s", UPLOAD_HOME, ip));

I'm also fine with debug.